### ISSUE TITLE
Fix to IP

### ISSUE THREAD

**Issue by jtagcat** [louislam/uptime-kuma#1327]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

API

### 🔖 Feature description

 - What happened: monitored host has 2 ISPs attached to it, one went down (planned)
 - Browsers accessing websites: first page load takes a bit longer, browser automatically switches over to secondary IP address (DNS)
 - What uptime-kuma is doing: screaming everything is down

### ✔️ Solution

 - What I'd expect uptime-kuma to do: (optional) 'jfyi one of the IPs listed in DNS is down, but overall, we are fine'
 - What I'd also expect uptime-kuma to do: (optionally) monitor all IPs listed in the DNS, not whichever it reached first.

### 📝 Additional Context

fucking let me use issue template without forced fields

**Comment by erwineverts** [comment]
I would agree that monitoring all IP's for a given DNS name should be able to be monitored.

Would the behavior be that if one of the hosts fails, the site is down?

**Comment by jtagcat** [comment]
Yea; kuma has only booleans OK (up) or NOK (down), there is no degraded status. Though, degraded might not be good either, as this could be a security issue, where it is definitely NOK/down.

**Comment by maxheadroom** [comment]
I have the same issue here. In my opinion the host which has multiple IPs schould not be considered down as long as at least one of the IPs can be reached. At max a warning could be issued.

**Comment by suominen** [comment]
It would be nice if Happy Eyeballs ([RFC 8305](https://datatracker.ietf.org/doc/html/rfc8305)) was an option. It seems this was already discussed in #57.

Would it be possible to make use of this happy-eyeballs package:
- https://github.com/zwhitchcox/happy-eyeballs
- https://www.npmjs.com/package/happy-eyeballs

Node.js still doesn't appear to have Happy Eyeballs (see: nodejs/node#41625).

**Comment by CommanderStorm** [comment]
You can select your IP family now
